src/components/package/edit-managed/managed-package-edit.js

```diff
@@ -241,35 +245,11 @@ class ManagedPackageEdit extends Component {
                 <DetailsViewSection
                   label={intl.formatMessage({ id: 'ui-eholdings.label.holdingStatus' })}
                 >
-                  <label
-                    data-test-eholdings-package-details-selected
-                    htmlFor="managed-package-details-toggle-switch"
-                  >
-                    { packageSelectionPending ? (
-                      <Icon icon="spinner-ellipsis" />
-                    ) : (
-                      <h4>
-                        {packageSelected ?
-                          (<FormattedMessage id="ui-eholdings.selected" />) :
-                          (<FormattedMessage id="ui-eholdings.notSelected" />)
-                        }
-                      </h4>
-                    )}
-                    <br />
-
-                    {((!packageSelected && !packageSelectionPending) ||
-                      (!this.props.model.isSelected && packageSelectionPending)) &&
-                      <Button
-                        type="button"
-                        buttonStyle="primary"
-                        disabled={packageSelectionPending}
-                        onClick={this.handleSelectionAction}
-                        data-test-eholdings-package-add-to-holdings-button
-                      >
-                        <FormattedMessage id="ui-eholdings.package.addToHoldings" />
-                      </Button>
-                    }
-                  </label>
+                  <SelectionStatus
+                    model={model}
+                    isPending={packageSelectionPending}
```

Comment:
Same here, we should be able to get rid of this, correct since it will be ignored by the `SelectionStatus` component.

Reply:
All set!